main.css

```
@@ -0,0 +1,369 @@
+body {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Poppins', 'Arial', 'Montserrat', sans-serif;
+    display: flex;
+    flex-direction: column;
+}
+
+.header {
+    width: 100%;
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    background-color: #000;
+    height: 60px;
+
+}
+
+.header__logo {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.header__nav {
+    height: 100%;
+    display: flex;
+}
+
+.nav__item {
+    height: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 0 20px;
+    border-right: 1px solid #565656;
+}
+
+.nav__item:first-child {
+    border-left: 1px solid #565656;
+}
+
+.nav__link,
+.nav__link--button {
+    text-decoration: none;
+    text-transform: uppercase;
+    color: #efefef;
+    font-size: 12px;
+    font-weight: bold;
+    letter-spacing: 0.5px;
+}
+
+.nav__wrapper {
+    flex: 1;
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+}
+
+.nav__form {
+    display: flex;
+}
+
+.nav__input {
+    border: none;
+    border-radius: 2px;
+}
+
+.nav__button {
+    background: none;
+    color: #fff;
+    border: none;
+    cursor: pointer;
+}
+
+.main {
+    text-align: justify;
+}
+
+.main__article {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.article__content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 60%;
+    padding: 2em 0;
+}
+
+.article__header {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    background-size: cover;
+    background-image: linear-gradient(to right, rgba(0, 0, 0, 1),
+            rgba(0, 0, 0, 0.95),
+            rgba(0, 0, 0, 0.65),
+            rgba(0, 0, 0, 0.45),
+            rgba(0, 0, 0, 0.4),
+            rgba(0, 0, 0, 0.45),
+            rgba(0, 0, 0, 0.65),
+            rgba(0, 0, 0, 0.95),
+            rgba(0, 0, 0, 1)),
+        url("bycicle-bg2.jpg");

```

```suggestion
        url('bycicle-bg2.jpg');

```

Hotaraste-te ce fel de ghilimele folosesti. Simple sau duble.